README.md

```diff
@@ -1,4 +1,12 @@
 # CanSat
+## Prerequisites
+ 1. You will need **libusb** <br>
+ check out [cargo-embed](https://crates.io/crates/cargo-embed) for instructions
+ 2. Add your board target:
+```
+rustup target add thumbv7em-none-eabihf
+```
+
```

Comment:
I guess this change was not supposed to be in this PR considering the existence of https://github.com/grupacosmo/cansat/pull/7